README.md

```diff
@@ -508,6 +508,33 @@ The warnings available are:
 
 ## Upgrade Guides
 
+### 8.x to 9.0
+
+This release brings the react-imgix API more in-line with that of imgix's rendering service.
+
+The largest change users will notice is that this project's component will no longer generate a default `fit=crop` parameter. This was original intention behind this was so that generated images would maintain aspect ratio when at least one of the dimensions were specified. However, the default imgix API behavior [sets `fit=clip`](https://docs.imgix.com/apis/url/size/fit#clip), which is now reflected in this project.
```

Comment:
> This was original intention behind this
☝️ I think you got two sentences mashed up here.

Reply:
It's that Friday brain 🙃